ms-common-auth/src/main/java/net/trajano/ms/authz/internal/CacheNames.java

```diff
@@ -8,6 +8,8 @@
 
     public static final String JWKS = Qualifiers.JWKS_CACHE;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "JWKS_CACHE"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)